templates/basic_answerfile.txt.j2

```diff
@@ -0,0 +1,111 @@
+OVESETUP_DB/port=int:{{ovirt_engine_setup_db_port}}
+OVESETUP_DB/filter=none:None
+OVESETUP_DB/restoreJobs=int:2
+OVESETUP_DB/securedHostValidation=bool:False
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_DB/secured=bool:False
+OVESETUP_DWH_DB/host=str:{{ovirt_engine_setup_dwh_db_host}}
+OVESETUP_DWH_DB/user=str:{{ovirt_engine_setup_dwh_db_user}}
+OVESETUP_DWH_DB/database=str:{{ovirt_engine_setup_dwh_db_name}}
+OVESETUP_DWH_DB/port=int:{{ovirt_engine_setup_dwh_db_port}}
+{% else %}
+OVESETUP_DWH_DB/secured=none:None
+OVESETUP_DWH_DB/host=none:None
+OVESETUP_DWH_DB/user=none:None
+OVESETUP_DWH_DB/password=none:None
+OVESETUP_DWH_DB/database=none:None
+OVESETUP_DWH_DB/port=none:None
+{% endif %}
+{% if ovirt_engine_setup_dwh_db_password is defined %}
+OVESETUP_DWH_DB/password=str:{{ovirt_engine_setup_dwh_db_password}}
+{% else %}
+OVESETUP_DWH_DB/password=none:None
+{% endif %}
+OVESETUP_DWH_DB/dumper=str:pg_custom
+OVESETUP_DWH_DB/filter=none:None
+OVESETUP_DWH_DB/restoreJobs=int:2
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_DB/securedHostValidation=bool:False
+{% else %}
+OVESETUP_DWH_DB/securedHostValidation=none:None
+{% endif %}
+OVESETUP_ENGINE_CORE/enable=bool:True
+OVESETUP_CORE/engineStop=none:None
+OVESETUP_SYSTEM/memCheckEnabled=bool:False
+OVESETUP_SYSTEM/nfsConfigEnabled=bool:False
+OVESETUP_PKI/organization=str:{{ ovirt_engine_setup_organization }}
+OVESETUP_PKI/renew=none:None
+{% if ovirt_engine_setup_configure_iso_domain is defined %}
+OVESETUP_CONFIG/isoDomainName=str:{{ ovirt_engine_setup_iso_domain_name }}
+OVESETUP_CONFIG/isoDomainACL=str:{{ ovirt_engine_setup_iso_domain_acl }}
+OVESETUP_CONFIG/isoDomainMountPoint=str:{{ ovirt_engine_setup_iso_domain_path }}
+OVESETUP_SYSTEM/nfsConfigEnabled=bool:True
+{% else %}
+OVESETUP_CONFIG/isoDomainName=none:None
+OVESETUP_CONFIG/isoDomainACL=none:None
+OVESETUP_CONFIG/isoDomainMountPoint=none:None
+OVESETUP_SYSTEM/nfsConfigEnabled=bool:False
+{% endif %}
+OVESETUP_CONFIG/engineHeapMax=str:1024M
+OVESETUP_CONFIG/ignoreVdsgroupInNotifier=none:None
+OVESETUP_CONFIG/adminPassword=str:{{ ovirt_engine_setup_admin_password }}
+OVESETUP_CONFIG/engineDbBackupDir=str:/var/lib/ovirt-engine/backups
+OVESETUP_CONFIG/engineHeapMin=str:1024M
+OVESETUP_CONFIG/imageioProxyConfig=bool:True
+OVESETUP_PROVISIONING/postgresProvisioningEnabled=bool:True
+OVESETUP_APACHE/configureRootRedirection=bool:True
+OVESETUP_APACHE/configureSsl=bool:True
+{% if ovirt_engine_setup_dwh_db_configure is defined and ovirt_engine_setup_dwh_db_configure %}
+OVESETUP_DWH_CORE/enable=bool:True
+{% else %}
+OVESETUP_DWH_CORE/enable=bool:False
+{% endif %}
+OVESETUP_DWH_CONFIG/scale=str:1
+OVESETUP_DWH_CONFIG/dwhDbBackupDir=str:/var/lib/ovirt-engine-dwh/backups
+OVESETUP_DWH_DB/restoreBackupLate=bool:True
+OVESETUP_DWH_DB/disconnectExistingDwh=none:None
+OVESETUP_DWH_DB/performBackup=none:None
```

Comment:
Better drop, same as OVESETUP_CORE/engineStop

Reply:
I don't know what is the default option here to set it explicitly. You mean I should set it to False?

Reply:
And is this option also related only to upgrade? I should probably move these options to some upgrade specific file.